deps/gtest/gtest.gyp

```diff
@@ -3,11 +3,22 @@
     {
       'target_name': 'gtest',
       'type': 'static_library',
-      'cflags': ['-Wno-missing-field-initializers'],
+      'cflags': [
+        '-Wno-missing-field-initializers',
+      ],
       'direct_dependent_settings': {
-        'include_dirs': ['include'],
+        'include_dirs': [
+          'include',
+        ],
       },
-      'include_dirs': ['.', 'include'],
+      'defines': [
+        'GTEST_LANG_CXX11=1',
```

Comment:
https://codereview.chromium.org/2174663002/. Maybe we should be bumping our vendored in version of `gtest`?

Reply:
I have that in the pipe line.